tests/ballerina-test/src/test/resources/test-src/statements/expression/ignore-values-negative.bal

```diff
@@ -3,13 +3,13 @@ function add (int x, int y) returns string {
     return result;
 }
 
-function checkAndAdd (int x, int y) returns (string, error|null) {
+function checkAndAdd (int x, int y) returns (string, error|()) {
```

Comment:
Can't we just return (string|error)

Reply:
Yes. Seems it's unnecessary to have two return types for this scenario. We can just use ```return (string|error)```. I have updated the PR accordingly. Thanks for the suggestion.